Updating endpoits (#323)
metaplex-foundation/developer-hub: src/pages/core-candy-machine/guides/create-a-core-candy-machine-with-hidden-settings.md

While setting up Umi, you can create new wallets for testing, import wallets from you filesystem or even use `walletAdapter` with a UI/frontend.
For this example, we will be creating a Keypair from a json file (wallet.json) containing a secret key.

We will be using the devnet Aura endpoint.

> To gain access to Metaplex Aura network on the Solana and Eclipse blockchains you can visit the Aura App for an endpoint and API key [here](https://aura-app.metaplex.com/).

Reviewer: ```suggestion
To gain access to the Metaplex Aura network on the Solana and Eclipse blockchains you can visit the Aura App for an endpoint and API key [here](https://aura-app.metaplex.com/).
```

```ts
import { createUmi } from "@metaplex-foundation/umi-bundle-defaults";
import { generateSigner, some, none, createSignerFromKeypair, signerIdentity, transactionBuilder, dateTime } from "@metaplex-foundation/umi";
import { mplCandyMachine as mplCoreCandyMachine } from '@metaplex-foundation/mpl-core-candy-machine';
import * as fs from 'fs';

> // We will be using Solana Devnet from Aura data network as endpoint, and loading the mplCoreCandyMachine plugin

Reviewer: ```suggestion
// We will be using Solana Devnet from the Metaplex Aura data network as the endpoint while also loading the `mplCoreCandyMachine()` plugin.
```
